src/server/graphics/platform_probe.cpp

```diff
@@ -14,6 +14,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include "mir/graphics/default_display_configuration_policy.h"
```

Comment:
I guess it is pre-existing. But we try to ensure that the first header included is the one that corresponds. (In this case `#include "platform_probe.h"`